src/offchainapi/tests/remote_benchmark.py

```diff
@@ -85,45 +85,46 @@ def run_server(my_configs_path, other_configs_path, num_of_commands=10, loop=Non
             'port': <int>,
         }
     '''
```

Comment:
I don't mind to comment these, but what's reason to comment them? seems should not cause issue, or it is configs needs to be updated? you probably can get new address by:
```

from diem import identifier
account_address, subaddress = identifier.decode_account(addr_str, "tlb" or "lbr")
print(identifier.encode_account(account_address, subaddress, 'tdm' or 'dm'))

```

Reply:
Commented out while i was trying to run unit tests only -- reverted it now